packages/office-addin-dev-settings/src/sideload.ts

```diff
@@ -237,36 +237,34 @@ async function getOutlookVersion(): Promise<string | undefined> {
   }
 }
 
-async function getOutlookExePath(): Promise<string> {
+async function getWXPOExePath(app: OfficeApp): Promise<string> {
```

Comment:
Don't use "wxpo" in the method name . . . it should be a general name for the general thing that it is doing (we don't want to have to change the name when there's an update)

Reply:
Got it. I will change it to "getOfficeExePath". How about do you think?